GVRf/Extensions/3DCursor/IODevices/gvrmobileiodevice/src/main/java/com/example/gvrmobileiodevice/WifiDirectBroadcastReceiver.java

```diff
@@ -0,0 +1,132 @@
+package com.example.gvrmobileiodevice;
+
+import android.content.BroadcastReceiver;
+import android.content.Context;
+import android.content.Intent;
+import android.net.wifi.ScanResult;
+import android.net.wifi.WifiManager;
+import android.net.wifi.p2p.WifiP2pConfig;
+import android.net.wifi.p2p.WifiP2pDevice;
+import android.net.wifi.p2p.WifiP2pDeviceList;
+import android.net.wifi.p2p.WifiP2pManager;
+import android.net.wifi.p2p.WifiP2pManager.ActionListener;
+import android.net.wifi.p2p.WifiP2pManager.Channel;
+import android.net.wifi.p2p.WifiP2pManager.PeerListListener;
+import android.util.Log;
+
+/**
+ * Created by v.nidamarthy on 8/29/16.
```

Comment:
We dont usually put author names in source files. Git blame is a good tool to find author for a file,